docs/internal.rst

```diff
@@ -32,7 +94,11 @@ For example,
     ti.init(print_ir=True)
 
     x = ti.var(ti.i32)
-    ti.root.dense(ti.i, 4).bitmasked(ti.i, 4).place(x)
+
+    S0 = ti.root
+    S1 = S0.dense(ti.i, 4)
+    S2 = S1.bitmasked(ti.i, 4)
+    S2.place(x)
```

Comment:
Can you add some comments to clarify who's cell, who's compoment?

Reply:
Actually, they are all SNodes.

Reply:
Then, what these SNodes represents? Please don't be too strict on this, or it's hard to understand for newcomers. We know that PRs should be small, the same to concepts.